.github/ISSUE_TEMPLATE/app_rendering_bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Application Card Rendering Bug Report
+about: Create a bug report for problems rendering Adaptive Cards in an Application
+title: "[Rendering] Bug Title Here"
+labels: Bug Triage-Needed Rendering
```

Comment:
We have previous labels that we should either rename to this or combine. Generally `Area-Renderers` for rendering, and `Area-Schema` for authoring. There are no doubt plenty of old issues with these labels though

Reply:
Great catch - I meant to go and find those labels before the PR...thanks for pointing them out!